### PR TITLE
nf-core modules update bowtie2/align

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/atacseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -239,7 +239,7 @@ manifest {
     description     = 'ATACSeq peak-calling and differential analysis pipeline.'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version         = '2.0'
+    version = '2.1dev'
     doi             = 'https://doi.org/10.5281/zenodo.2634132'
```
Comment (drpatelh) [inline nextflow.config:243]:
```suggestion
    version         = '2.1dev'
    doi             = 'https://doi.org/10.5281/zenodo.2634132'
```

Comment (maxulysse) [reply nextflow.config:243]: tools change the spacing for me

**Comment by drpatelh** [comment]
Thanks @maxulysse !

**Review by drpatelh — APPROVED** [review]
🚀

**Pull request merged**